Small how to uninstall certbot-auto (#7648)
certbot/certbot: certbot/docs/install.rst

from your web server OS and putting others in a python virtual environment. You can
download and run it as follows::

>   cd /tmp

Reviewer: I think this can probably be omitted. If we're worried about security, we could do something like `cd $(mktemp -d)`, but I think it's overkill and distracts people from the important parts of the instructions.

Author: it's just that you don't want to download the file in a random dir but then i realized that the file is moved anyway so yes it is quite useless

  wget https://dl.eff.org/certbot-auto
  sudo mv certbot-auto /usr/local/bin/certbot-auto
  sudo chown root /usr/local/bin/certbot-auto
  sudo chmod 0755 /usr/local/bin/certbot-auto
  /usr/local/bin/certbot-auto --help

> To remove certbot-auto, just delete it and the files it places in /opt (aka /opt/eff.org), along with any cronjob you may have created.

Reviewer: ```suggestion
To remove certbot-auto, just delete it and the files it places under /opt/eff.org, along with any cronjob or systemd timer you may have created.
```
